pkpdapp/pkpdapp/templates/dataset_detail.html

```diff
@@ -6,10 +6,35 @@
 
 {% extends "base/base.html" %}
 {% load static %}
+{% load markdownify %}
 
 {% block main_content %}
 <p></p>
 <h2>Dataset {{dataset.name}}</h2>
-<p>{{dataset.description}}</p>
+(<a href="{% url 'dataset-update' dataset.id %}"> Update </a>)
+<p>{{dataset.description|markdownify}}</p>
+<h3>Biomarker Types</h2>
+<ul>
+{% for type in biomarker_types %}
```

Comment:
Do we want to display all the data or just a subset of it? Just think showing (say) 1000 rows might make display a bit better. (I know we'll probably want to do this in a nicer markdowny way later.)

Reply:
yup, good idea

Reply:
I've paginated this, so you can click through multiple pages each of length 20